lib/config.js

```diff
@@ -140,8 +141,31 @@ function getLocalConfig(thisConfig, directory) {
         config = ConfigOps.merge(localConfig, config);
     }
 
-    // Use the personal config file if there are no other local config files found.
-    return found || thisConfig.useSpecificConfig ? config : ConfigOps.merge(config, getPersonalConfig());
+    if (found || thisConfig.useSpecificConfig) {
+        return config;
+    } else {
+
+        /**
```

Comment:
Ack, you're right. I was misremembering the repo guidelines for non JSDoc
multiple line comments. Will fix later tonight.
On Jun 22, 2016 7:00 PM, "Ian VanSchooten" <notifications@github.com> wrote:

> In lib/config.js
> <https://github.com/eslint/eslint/pull/6384#discussion_r68156797>:
>
> > @@ -140,8 +141,31 @@ function getLocalConfig(thisConfig, directory) {
> >          config = ConfigOps.merge(localConfig, config);
> >      }
> >
> > -    // Use the personal config file if there are no other local config files found.
> > -    return found || thisConfig.useSpecificConfig ? config : ConfigOps.merge(config, getPersonalConfig());
> > +    if (found || thisConfig.useSpecificConfig) {
> > +        return config;
> > +    } else {
> > +
> > +        /**
>
> Technically, I think /** is JSDoc notation, isn't it? Should this perhaps
> just be /*? Maybe it doesn't matter.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/eslint/eslint/pull/6384/files/94bdc2000dfe96c13fe9b8cb9260364753f2dfff#r68156797>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AARWegqLLQzvcBtMqekqxJdSD_K42Dzjks5qOcyPgaJpZM4IzpJl>
> .
>